packages/ui/src/components/editor/properties/variant/index.tsx

```diff
@@ -0,0 +1,325 @@
+/*
+CPAL-1.0 License
+
+The contents of this file are subject to the Common Public Attribution License
+Version 1.0. (the "License"); you may not use this file except in compliance
+with the License. You may obtain a copy of the License at
+https://github.com/EtherealEngine/etherealengine/blob/dev/LICENSE.
+The License is based on the Mozilla Public License Version 1.1, but Sections 14
+and 15 have been added to cover use of software over a computer network and 
+provide for limited attribution for the Original Developer. In addition, 
+Exhibit A has been modified to be consistent with Exhibit B.
+
+Software distributed under the License is distributed on an "AS IS" basis,
+WITHOUT WARRANTY OF ANY KIND, either express or implied. See the License for the
+specific language governing rights and limitations under the License.
+
+The Original Code is Ethereal Engine.
+
+The Original Developer is the Initial Developer. The Initial Developer of the
+Original Code is the Ethereal Engine team.
+
+All portions of the code written by the Ethereal Engine team are Copyright © 2021-2023 
+Ethereal Engine. All Rights Reserved.
+*/
+
+import React, { useEffect } from 'react'
+import { useTranslation } from 'react-i18next'
+import { MdDeblur } from 'react-icons/md'
+
+import { AssetExt } from '@etherealengine/common/src/constants/AssetType'
+import { getOptionalMutableComponent, useComponent } from '@etherealengine/ecs/src/ComponentFunctions'
+import { Entity, UndefinedEntity } from '@etherealengine/ecs/src/Entity'
+import Center from '@etherealengine/editor/src/components/layout/Center'
+import PaginatedList from '@etherealengine/editor/src/components/layout/PaginatedList'
+import {
+  EditorComponentType,
+  commitProperties,
+  commitProperty
+} from '@etherealengine/editor/src/components/properties/Util'
+import { AssetLoader } from '@etherealengine/engine/src/assets/classes/AssetLoader'
+import { loadResource } from '@etherealengine/engine/src/assets/functions/resourceLoaderFunctions'
+import { ModelComponent } from '@etherealengine/engine/src/scene/components/ModelComponent'
+import { Heuristic, VariantComponent, VariantLevel } from '@etherealengine/engine/src/scene/components/VariantComponent'
+import { State, getState, useHookstate } from '@etherealengine/hyperflux'
+import {
+  ResourceManager,
+  ResourceState,
+  ResourceStatus,
+  ResourceType
+} from '@etherealengine/spatial/src/resources/ResourceState'
+import { Button } from '@mui/material'
```

Comment:
there is a mui button present here
![image](https://github.com/EtherealEngine/etherealengine/assets/55396651/ff58e34c-aa72-4eb6-8711-6170fe5c15a8)

change it to tailwind button